src/pages/taskQueue/DistroOption.tsx

```diff
@@ -18,15 +20,17 @@ export const DistroOption: React.VFC<DistroOptionProps> = ({
 }) => {
   const { hostCount, id, taskCount } = option;
   return (
-    <OptionWrapper onClick={() => onClick(option)}>
-      <StyledBadge>{`${option.taskCount} ${
-        taskCount === 1 ? "TASK" : "TASKS"
-      }`}</StyledBadge>
-      <StyledBadge>{`${hostCount} ${
-        hostCount === 1 ? "HOST" : "HOSTS"
-      }`}</StyledBadge>
-      <DistroName>{id}</DistroName>
-    </OptionWrapper>
+    <Link to={getTaskQueueRoute(id)} onClick={() => onClick(option)}>
+      <OptionWrapper>
+        <StyledBadge>{`${option.taskCount} ${
+          taskCount === 1 ? "TASK" : "TASKS"
+        }`}</StyledBadge>
+        <StyledBadge>{`${hostCount} ${
+          hostCount === 1 ? "HOST" : "HOSTS"
```

Comment:
[nit] Badges apply text-transform so we should just use normal casing

```suggestion
          taskCount === 1 ? "Task" : "Tasks"
        }`}</StyledBadge>
        <StyledBadge>{`${hostCount} ${
          hostCount === 1 ? "Host" : "Hosts"
```